src/content/recipes/@emotion/styled.md

```diff
@@ -29,7 +29,25 @@ If you’d like to see the example code of this recipe, check out the [example r
 
 ![Completed Emotion example with theme switcher](https://user-images.githubusercontent.com/18172605/208312563-875ca3b0-e7bc-4401-a445-4553b48068ed.gif)
 
-## Install `@storybook/addon-styling`
+## Auto-config
+
+As of version 1.1, `@storybook/addon-styling` offers a codemod for to automatically configure your storybook with Emotion.
+
+To try it out, run the following commands:
+
+```shell
+# Install the addon
+yarn add -D @storybook/addon-styling
+
+# Run the postinstall script from the root of your project
```

Comment:
Will this happen automatically if they haven't disabled postinstall scripts? Worth mentioning that, and couching this instruction in "If it didn't run automatically..."?

Reply:
It won't be able to run automatically until we update the CLI :(

Reply:
Sorry for the lack of clarity, I was referring to `postinstall` as documented here: https://classic.yarnpkg.com/en/docs/package-json#toc-scripts